contents/docs/libraries/ios/index.mdx

```diff
@@ -36,6 +36,102 @@ PostHog supports the following Apple platforms:
 
 <IOSSendEvents />
 
+## Autocapture 
+
+PostHog autocapture automatically tracks the following events for you:
+
+-   **Application Opened** - when the app is opened from a closed state or when the app comes to the foreground (e.g. from the app switcher)
+-   **Application Backgrounded** - when the app is sent to the background by the user
+-   **Application Installed** - when the app is installed.
+-   **Application Updated** - when the app is updated.
+-   **$screen** - when the user navigates (if using `UIViewController`)
+-   **$autocapture** - interactions with elements in a screen (if using `UIKit`)
+
+> 🚧 **Note:** `$autocapture` is currently supported only in UIKit and remains in an experimental state, with the feature disabled by default. SwiftUI support is planned for an upcoming release.
+
+### Autocapture configuration
+
+You can enable or disable Autocapture through the `PostHogConfig` object.
+
+```swift
+let POSTHOG_API_KEY = "<ph_project_api_key>"
+// usually 'https://us.i.posthog.com' or 'https://eu.i.posthog.com'
+let POSTHOG_HOST = "https://us.i.posthog.com"
+
+let config = PostHogConfig(apiKey: POSTHOG_API_KEY, host: POSTHOG_HOST)
+
+// capture screen views
+config.captureScreenViews = true 
+// capture application lifecycle events
+config.captureApplicationLifecycleEvents = true
+// enable $autocapture events
+config.autocapture = true // TIP: This is false by default
+// for $autocapture events, capture text input changes and edits
+config.autocaptureConfig.captureTextEdits = true 
+// for $autocapture events, capture gestures such as swipes and taps
+config.autocaptureConfig.captureGestures = true
+// for $autocapture events, capture control actions such as button presses
+config.autocaptureConfig.captureControlActions = true
+// for $autocapture events, capture control values or text (title for UIButton, text for UITextField, etc.)
+config.autocaptureConfig.captureValues = true 
```

Comment:
same as https://github.com/PostHog/posthog.com/pull/9783/files#r1829409919
its not really obvious what capture"Values" means.

Reply:
I think a config for "Texts" in general makes sense because of PII.
Values in general is part of the feature, either that be a boolean or an int.
JS does not have granular configurations, you either enable or disable autocapture.
JS-Lite has captureTouches as part of the config along with captureLifecycleEvents and captureScreens, which is ok since the booleans are within the autocapture config.
IMO the best configs we should give is which controls we want to enable or disable, this can be done after, and now is all enabled or all disabled, an example would be `captureControls[ComponentType]`, so I can disable TextInput for example.
A config for sensitive data should be possible, so texts are not autocaptured by default.
TLDR I'd remove all those booleans and would only offer a config for sensitive data (texts in general), most likely because of text inputs.

Reply:
JS has `capture_copied_text` because of the sensitive issue, otherwise would always be enabled as well